### ISSUE TITLE
Unable to install pixeloe-0.0.9

### ISSUE THREAD

**Issue by Gileanxxx** [cubiq/ComfyUI_essentials#33]
![image](https://github.com/cubiq/ComfyUI_essentials/assets/163095612/5b1e5a22-2bdb-4270-82bd-5e7351da4b08)
My comfyui version is: ComfyUI: 2171[4f63ee](2024-05-10)
Manager: V2.30.1
Can this be solved?

**Comment by cubiq** [comment]
you need to install pixeloe...

**Comment by Gileanxxx** [comment]
> you need to install pixeloe...

Yes, I came to ask for help because I could not install pixeloe. This error occurred when I used the "Python -m pip install pixeloe" command in power shell and CMD. Is this an individual example? I used " The official integration package downloaded from https://github.com/comfyanonymous/ComfyUI/releases/download/latest/ComfyUI_windows_portable_nvidia_cu121_or_cpu.7z” was just downloaded today.
